### PR TITLE
Enable changing display type of weights (factor or percentage)

### PR DESCRIPTION
**Pull request by Jonas-Sander**
Adds changing the display type of weights (as factor or as percent) in the term settings.

[Changing weight display type.webm](https://github.com/user-attachments/assets/131389f5-771e-4451-ae5b-fac602811370)

| Before | After |
|--------|--------|
| before | after |

The numbers are currently rounded in the weight overview, I'll fix this in another PR.

### CODE REVIEW

**Review by Sharezone-Admin — APPROVED** [review]
(Jonas S here, approving with this account so that I can merge via merge queue)

**Comment by Jonas-Sander** [comment]
Nvm, would need to add the account to code owners. Merging with override.

**Pull request merged**
